pkg/apis/experimental/types.go

```diff
@@ -540,3 +540,64 @@ type IngressBackend struct {
 	// Specifies the protocol of the referenced service.
 	Protocol api.Protocol `json:"protocol,omitempty"`
 }
+
+// DerivedNodeMetricsList contains derived metrics for a single node. It holds
+// derived metrics in windows about the node itself and all of its system
+// containers.
```

Comment:
What are system containers? It is a new concept that this PR is introducing. Shall we instead say `all the cgroups created and managed by kubernetes on the node`?